### ISSUE TITLE
Merge in mathcomp proper

### ISSUE THREAD

**Issue by gares** [arthuraa/coq-void#1]
At the last math comp meeting we discussed about this code, and we believe it makes sense to merge it in mathcomp. Would you agree?

**Comment by arthuraa** [comment]
Yes, I think it would be great. Do you mean adding the instances at the corresponding files in ssreflect, or as a separate project under the mathcomp page?

**Comment by gares** [comment]
I'm talking about a pull request github.com/math-comp/math-comp

**Comment by arthuraa** [comment]
That sounds good. I'll put something up.

**Comment by anton-trunov** [comment]
This is done in https://github.com/math-comp/math-comp/pull/393
